pwndbg/abi.py

```diff
@@ -150,3 +162,7 @@ def caller(*args, **kwargs):
         return caller
 
     return decorator
+
+
+# XXX: Update when starting the gdb to show warning message for non-Linux ABI user.
```

Comment:
Can we just do that? Like `if non_linux_abi: warn_not_linux()` or sth

Reply:
The the abi information is updated by `update()`. We have to update it at least one time.
The other approach is change `linux` variable to a getter of module and calculate each time (since the information fetching from gdb is not an expensive cost). But I don't know how to set a getter of module in Python.

Reply:
We can leave it as it is; just update the comment (we dont want '###' ;P)